src/StylesProvider.js

```diff
@@ -8,7 +8,6 @@ import themeLight from './styles/themeLight';
 
 const themeNameToObject = {
   'default': themeLight,
-  'light': themeLight,
```

Comment:
@saketkumar95 consider this commit https://github.com/zulip/zulip-mobile/commit/a21bf82446594450217a850b73e42882aae21aa2, through probably this change shouldn't cause a problem with the production app...